pkg/controller/core/queue_controller.go

```diff
@@ -131,23 +132,23 @@ type queuedWorkloadHandler struct{}
 func (h *queuedWorkloadHandler) Create(e event.CreateEvent, q workqueue.RateLimitingInterface) {
 	qw := e.Object.(*kueue.QueuedWorkload)
 	if qw.Spec.Admission == nil {
-		q.Add(requestForQueueStatus(qw))
+		q.AddAfter(requestForQueueStatus(qw), time.Second*1)
```

Comment:
just `time.Second`

Also, can you do the same for the clusterqueue_controller?

Reply:
I would something higher than a second, something higher would be more resilient and doesn't really impact the user experience.